### PR TITLE
[Enhancement] Prioritize tasks in ThreadPool

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

### CODE REVIEW

**Review by decster — DISMISSED** [review]

File: be/src/util/threadpool.h [hunk 1]
```diff
@@ -156,6 +157,12 @@ class ThreadPoolBuilder {
 //    thread_pool->SubmitFunc(std::bind(&Func, 10));
 class ThreadPool {
 public:
+    enum Priority {
+        LOW_PRIORITY,
```
Comment (imay) [inline be/src/util/threadpool.h:161]:
```suggestion
        LOW_PRIORITY = 0,
```

Comment (sduzh) [reply be/src/util/threadpool.h:161]: 0 is the default starting value.
https://stackoverflow.com/questions/34811486/do-c-enums-start-at-0

Comment (imay) [reply be/src/util/threadpool.h:161]: sure, I'd like make it clear, easy to understand

Comment (sduzh) [reply be/src/util/threadpool.h]: updated

**Pull request merged**